src/librbd/image/CreateRequest.h

```diff
@@ -0,0 +1,170 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#ifndef CEPH_LIBRBD_CREATEREQUEST_H
```

Comment:
Minor: extra line break spacing between lines 2 + 3 and use "_" between words (CREATE_REQUEST)